loader/src/injector/module.h

```diff
@@ -157,11 +160,22 @@ struct rezygisk_module {
   bool unload;
 };
 
-void rezygisk_module_call_on_load(struct rezygisk_module *m, void *env) {
+/* What follows are function definitions to be included wherever necessary.
+    As a reminder for best C practices, a function body should not be in a header
+    since they lead to ODR violations, resulting in UB since the compiled code *can* have duplicate defintions.
+    Therefore, we have only ONE of two choices:
+        1. Put the function declarations here and their respective definitions in a separate .c file;
+        2. Inline these function definitions in the header so as to allow multiple definitions.
+
+    Doing otherwise, clang-tidy throws this warning:
+    https://clang.llvm.org/extra/clang-tidy/checks/misc/definitions-in-headers.html
+*/
```

Comment:
```suggestion
/* INFO: What follows are function definitions to be included wherever necessary.
                  As a reminder for best C practices, a function body should not be in a header
                  since they lead to ODR violations, resulting in UB since the compiled code *can* have duplicate defintions.
                  Therefore, we have only ONE of two choices:
                    1. Put the function declarations here and their respective definitions in a separate .c file;
                    2. Inline these function definitions in the header so as to allow multiple definitions.

                Doing otherwise, clang-tidy throws this warning: https://clang.llvm.org/extra/clang-tidy/checks/misc/definitions-in-headers.html
*/
```